docs/requirements.md

```diff
@@ -96,7 +96,7 @@ The storage requirements recommended on the following table **depend mainly on t
 Codacy requires a database server to persist data that must satisfy the following requirements:
 
 -   The infrastructure hosting the database server must be provisioned with the hardware requirements described below
--   The DBMS server must be [PostgreSQL](https://www.postgresql.org/) **version 10.20**
+-   The DBMS server must be [PostgreSQL](https://www.postgresql.org/) **version 11.16**
```

Comment:
The latest PostgreSQL 11.* version is now 11.18 (https://www.postgresql.org/support/versioning/). Should we require that version instead?